core/lexicon/en/login.inc.php

```diff
@@ -38,7 +38,7 @@
 $_lang['login_greeting_afternoon'] = '<strong>Good afternoon</strong>, welcome back!';
 $_lang['login_greeting_evening'] = '<strong>Good evening</strong>, welcome back!';
 $_lang['login_greeting_night'] = '<strong>Good night</strong>, welcome back!';
-$_lang['login_note'] = 'Feel free to login anytime.';
+$_lang['login_note'] = 'Please login to access the manager.';
```

Comment:
“Manager“ should be capitalized, and I believe it should be "log in" also.

Reply:
we did a vote on "login" vs "log in" and decided both work. I'm fine with both.

Reply:
`Manager` fixed. Ben, @Mark-H and @gpsietzema voted for `login` instead `log in`.

Reply:
Grammatically, "log in" is a verb. A "login" is a noun or adjective.

After you enter your login to log in, you've logged in, not loginned. You can ask a friend to log you in, but it'd sound weird if you tried to get them to login you.

:P

Reply:
Sounds ok for me. Updated!